ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -85,7 +85,7 @@ function CollectionsTable({
     const onSearchInputChange = useMemo(
         () =>
             debounce(
-                (value: string) => setSearchFilter({ Collection: value }),
+                (value: string) => setSearchFilter({ 'Collection Name': value }),
```

Comment:
This is an unrelated change, but not worth a separate PR.

The BE team is using `Collection Name` instead of `Collection` for this search label.